examples/06-distributed-post/00-distributed_total_disp.py

```diff
@@ -1,94 +1,115 @@
-"""
-.. _ref_distributed_total_disp:
```

Comment:
You lost the reference here but not in the other examples, is that on purpose?

Reply:
No it wasn't intentional. I will add it back.